providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/targeting/Operator.java

```diff
@@ -38,8 +40,12 @@ public Operator() {
      */
     public Object apply(final String flagKey, final String targetingRule, final EvaluationContext ctx)
             throws TargetingRuleException {
-        final Map<String, String> flagdProperties = new HashMap<>();
+        final Map<String, Object> flagdProperties = new HashMap<>();
```

Comment:
I'm not sure I follow, - the `unixTimestamp` is a long - are you suggesting we'd convert it to a string and back? I don't think that parsing is worth the small amount of type safety achieved.

Reply:
actually we have a bug if we do not store this as a string 

After storage in map, value extraction happens at `extractSubPropertyFromFlagd` [1]. If we do not store a String, then the value will be always null. 

We also need to add a test case for the fractional evaluator to use the timestamp. This is because that's the only place we extract these values right now. 

[1] - https://github.com/open-feature/java-sdk-contrib/pull/512/files#diff-d6fbc93c9666fdf8ee1e842ca02384dfc839756320162f7978be207739a3f095R91-R92

Reply:
This seems more like a problem with the validation in `extractSubPropertyFromFlagd`, than a problem with how it's stored. Can we just modify the filter call: `.filter(value -> value instanceof String)` to be instanceof Object?

I still think parsing back and forth from string is a bad idea. :thinking:

Reply:
@toddbaert Sorry I confused myself about the usage. 

Jsonlogic engine can also expand variable references so yes the issue is with `extractSubPropertyFromFlagd `. so the fix should go there.